decidim-core/app/views/decidim/newsletter_mailer/newsletter.html.erb

```diff
@@ -1,16 +1,28 @@
 <%= decidim_sanitize_newsletter cell.to_s %>
 
 <% content_for :note do %>
-  <%== t ".note", organization_name: h(@organization.name), link: decidim.notifications_settings_url(host: @organization.host) %>
+  <% if @newsletter.try(:draft?) %>
```

Comment:
for readability purposes maybe those 3 if staments could be extracted to stand alone methods like: `note_url`, `unsubscribe_url` ?

Reply:
Done at 4d502270032351170086778c0197e40c7293a20f.